doc/source/user_guide/timeseries.rst

```diff
@@ -2220,6 +2220,22 @@ you can use the ``tz_convert`` method.
 
    rng_pytz.tz_convert('US/Eastern')
 
+.. versionadded:: 1.1.0
+
+For ambiguous times, pandas supports explicitly specifying the fold argument.
```

Comment:
Could you mention in this description that we still recommend using `tz_localize` for more flexibility when handling ambiguous times?

Reply:
Done.